app/assets/javascripts/application.js

```diff
@@ -15,4 +15,21 @@
 //= require fullcalendar
 //= require listjs
 //= require daterangepicker
+//= require core.js/core.js
 //= require_tree .
+
+(function($) {
```

Comment:
Not sure how crucial this is, but maybe worth moving this stuff into it's own file? `pubsub.js` or something?